enh(mc): Great Reboot - network-firewalls-fortinet-fortigate-snmp (#2871)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-fortinet-fortigate-snmp.md

| CRITICALUSAGEFREE        | Thresholds                                                                                  |                   |             |
| WARNINGUSAGEPRCT         | Thresholds                                                                                  |                   |             |
| CRITICALUSAGEPRCT        | Thresholds                                                                                  |                   |             |
| EXTRAOPTIONS             | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |

</TabItem>
<TabItem value="SDWan" label="SDWan">

| Macro              | Description                                                                                                                                                                     | Valeur par défaut  | Obligatoire |
|:-------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-------------------|:-----------:|

> | FILTERVDOMAIN      | Filter sd-wan links by vdom name (can be a regexp)                                                                                                                              |                    |             |

Author: ```suggestion
| FILTERVDOMAIN      | Filter sd-wan links by vdom name (can be a regexp)                                                                                                                              |                    |             |
```
```suggestion
| FILTERVDOMAIN      | Define which virtual domains should be monitored based on their names. This option will be treated as a regular expression.                                                                                                                              |                    |             |
```

> | FILTERLINKNAME     | Filter sd-wan links by name (can be a regexp)                                                                                                                                   |                    |             |

Author: ```suggestion
| FILTERLINKNAME     | Define which SD-WAN links should be monitored based on their names. This option will be treated as a regular expression.                                                                                                                                   |                    |             |
```

> | FILTERLINKID       | Filter sd-wan links by ID (can be a regexp)                                                                                                                                     |                    |             |

Author: ```suggestion
| FILTERLINKID       | Define which SD-WAN links should be monitored based on their IDs. This option will be treated as a regular expression.                                                                                                                                     |                    |             |
```

| WARNINGIN     | Thresholds                                                                                  |                   |             |
| CRITICALIN    | Thresholds                                                                                  |                   |             |
| WARNINGOUT    | Thresholds                                                                                  |                   |             |
| CRITICALOUT   | Thresholds                                                                                  |                   |             |
| EXTRAOPTIONS  | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) |                   |             |

</TabItem>
<TabItem value="VPN-Global" label="VPN-Global">

| Macro              | Description                                                                                 | Valeur par défaut | Obligatoire |
|:-------------------|:--------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTER             | Filter name with regexp. Can be ('vdomain', 'vpn')                                          | .*                |             |

Author: ```suggestion
| FILTER             | Define which VPNs should be monitored based on their names. This option will be treated as a regular expression.                                          | .*                |             |
```

> | VDOMAIN            | Filter name with regexp. Can be ('vdomain', 'vpn')                                          |                   |             |

Author: ```suggestion
| VDOMAIN            | Define which virtual domains should be monitored based on their names. This option will be treated as a regular expression.                                          |                   |             |
```

| CRITICALTUNNELS    | Warning on counters. Can be ('users', 'sessions', 'tunnels', 'traffic-in', 'traffic-out')   |                   |             |
| WARNINGUSERS       | Warning on counters. Can be ('users', 'sessions', 'tunnels', 'traffic-in', 'traffic-out')   |                   |             |
| CRITICALUSERS      | Warning on counters. Can be ('users', 'sessions', 'tunnels', 'traffic-in', 'traffic-out')   |                   |             |
| EXTRAOPTIONS       | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles) | --verbose         |             |

</TabItem>
<TabItem value="Vdom-Usage" label="Vdom-Usage">

| Macro                    | Description                                                                                                                                | Valeur par défaut | Obligatoire |
|:-------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTERVDOMAIN            | Filter by virtual domain name (can be a regexp)                                                                                            |                   |             |

Author: ```suggestion
| FILTERVDOMAIN            | Define which virtual domains should be monitored based on their names. This option will be treated as a regular expression.                                                                                            |                   |             |
```

| WARNINGTRAFFICOUT        | Thresholds                                                                                                                                 |                   |             |
| CRITICALTRAFFICOUT       | Thresholds                                                                                                                                 |                   |             |
| EXTRAOPTIONS             | Any extra option you may want to add to the command (e.g. a --verbose flag). Toutes les options sont listées [ici](#options-disponibles)                                                | --verbose         |             |

</TabItem>
<TabItem value="Virus" label="Virus">

| Macro                     | Description                                                                                 | Valeur par défaut | Obligatoire |
|:--------------------------|:--------------------------------------------------------------------------------------------|:------------------|:-----------:|

> | FILTERNAME                | Filter virtual domain name (can be a regexp)                                                |                   |             |

Author: ```suggestion
| FILTERNAME                | Define which virtual domains should be monitored based on their names. This option will be treated as a regular expression.                                                |                   |             |
```

|:-----------|:----------------------------------|
| --warning  | Warning threshold in percent.     |
| --critical | Critical threshold in percent.    |

</TabItem>
<TabItem value="Hardware" label="Hardware">

| Option               | Description                                                                                                                                                                                                          |
|:---------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| --component          | Which component to check (default: '.*'). Can be: 'sensors'.                                                                                                                                                         |
| --add-name-instance  | Add literal description for instance value (used in filter, and threshold options).                                                                                                                                  |

> | --filter             | Exclude some parts (comma separated list) You can also exclude items from specific instances: --filter=sensors,1                                                                                                     |

Reviewer: ```suggestion
| --filter             | Exclude some parts (comma separated list). You can also exclude items from specific instances: --filter=sensors,1                                                                                                     |
```

| --statefile-format       | Define the format used to store the cache. Available formats: 'dumper', 'storable', 'json' (default).                                                                                                                                         |
| --statefile-key          | Define the key to encrypt/decrypt the cache.                                                                                                                                                                                                  |
| --statefile-cipher       | Define the cipher algorithm to encrypt the cache (default: 'AES').                                                                                                                                                                            |
| --warning-* --critical-* | Thresholds. Can be: 'intrusions-detected', 'intrusions-blocked', 'crit-sev-detections', 'high-sev-detections', 'med-sev-detections', 'low-sev-detections', 'info-sev-detections', 'signature-detections', 'anomaly-detections'.               |

> | --filter-name            | Filter virtual domain name (can be a regexp).                                                                                                                                                                                                 |

Reviewer: unclear

> | --filter-name            | Filter virtual domain name (can be a regexp).                                                                                                                                                                                                 |

Author: ```suggestion
| --filter-name            | Define which virtual domains should be monitored based on their names. This option will be treated as a regular expression.                                                                                                                                                                                                 |
```

| --statefile-concat-cwd   | If used with the '--statefile-dir' option, the latter's value will be used as a sub-directory of the current working directory. Useful on Windows when the plugin is compiled, as the file system and permissions are different from Linux.   |
| --statefile-format       | Define the format used to store the cache. Available formats: 'dumper', 'storable', 'json' (default).                                                                                                                                         |
| --statefile-key          | Define the key to encrypt/decrypt the cache.                                                                                                                                                                                                  |
| --statefile-cipher       | Define the cipher algorithm to encrypt the cache (default: 'AES').                                                                                                                                                                            |

> | --filter-id              | Filter sd-wan links by ID (can be a regexp).                                                                                                                                                                                                  |

Author: ```suggestion
| --filter-id              | Define which SD-WAN links should be monitored based on their IDs. This option will be treated as a regular expression.                                                                                                                                                                                                  |
```

> | --filter-name            | Filter sd-wan links by name (can be a regexp).                                                                                                                                                                                                |

Author: ```suggestion
| --filter-name            | Define which SD-WAN links should be monitored based on their names. This option will be treated as a regular expression.                                                                                                                                                                                                |
```

> | --filter-vdom            | Filter sd-wan links by vdom name (can be a regexp).                                                                                                                                                                                           |

Author: ```suggestion
| --filter-vdom            | Define which virtual domains should be monitored based on their names. This option will be treated as a regular expression.                                                                                                                                                                                           |
```

| --nagvis-perfdata                               | Display traffic perfdata to be compatible with nagvis widget.                                                                                                                                                                                                                              |
| --interface                                     | Set the interface (number expected) example: 1,2,... (empty means 'check all interfaces').                                                                                                                                                                                                 |
| --name                                          | Allows you to define the interface (in option --interface) byname instead of OID index. The name matching mode supports regular expressions.                                                                                                                                               |

> | --speed                                         | Set interface speed for incoming/outgoing traffic (in Mb).                                                                                                                                                                                                                                 |

Reviewer: unclear

> | --speed-in                                      | Set interface speed for incoming traffic (in Mb).                                                                                                                                                                                                                                          |

Reviewer: unclear

> | --speed-out                                     | Set interface speed for outgoing traffic (in Mb).                                                                                                                                                                                                                                          |

Reviewer: unclear

> | --force-counters32                              | Force to use 32 bits counters (even in snmp v2c and v3). Should be used when 64 bits counters are buggy.                                                                                                                                                                                   |

Reviewer: ```suggestion
| --force-counters32                              | Force to use 32-bit counters (even in snmp v2c and v3). Should be used when 64-bit counters are buggy.                                                                                                                                                                                   |
```

| --oid-display                                   | Define the OID that will be used to name the interfaces (default: ifName) (values: ifDesc, ifAlias, ifName, IpAddr, Vdom).                                                                                                                                                                 |
| --oid-extra-display                             | Add an OID to display.                                                                                                                                                                                                                                                                     |
| --display-transform-src --display-transform-dst | Modify the interface name displayed by using a regular expression.  Example: adding --display-transform-src='eth' --display-transform-dst='ens' will replace all occurrences of 'eth' with 'ens'                                                                                           |

> | --show-cache                                    | Display cache interface datas.                                                                                                                                                                                                                                                             |

Reviewer: ```suggestion
| --show-cache                                    | Display cache interface data.                                                                                                                                                                                                                                                             |
```

| --units-errors                                  | Units of thresholds for errors/discards (default: 'percent\_delta') ('percent\_delta', 'percent', 'delta', 'deltaps', 'counter').                                                                                                                                                          |
| --units-cast                                    | Units of thresholds for communication types (default: 'percent\_delta') ('percent\_delta', 'percent', 'delta', 'deltaps', 'counter').                                                                                                                                                      |
| --nagvis-perfdata                               | Display traffic perfdata to be compatible with nagvis widget.                                                                                                                                                                                                                              |

> | --interface                                     | Set the interface (number expected) example: 1,2,... (empty means 'check all interfaces').                                                                                                                                                                                                 |

Reviewer: ```suggestion
| --interface                                     | Define which interfaces to check (number expected). Example: 1,2... (empty means 'check all interfaces').                                                                                                                                                                                                 |
```

> | --name                                          | Allows you to define the interface (in option --interface) byname instead of OID index. The name matching mode supports regular expressions.                                                                                                                                               |

Reviewer: ```suggestion
| --name                                          | Allows you to define the interface (in option --interface) by name instead of OID index. The name matching mode supports regular expressions.                                                                                                                                               |
```

> | --speed                                         | Set interface speed for incoming/outgoing traffic (in Mb).                                                                                                                                                                                                                                 |

Reviewer: unclear

> | --speed-in                                      | Set interface speed for incoming traffic (in Mb).                                                                                                                                                                                                                                          |

Reviewer: unclear

| --speed-out                                     | Set interface speed for outgoing traffic (in Mb).                                                                                                                                                                                                                                          |

> | --force-counters32                              | Force to use 32 bits counters (even in snmp v2c and v3). Should be used when 64 bits counters are buggy.                                                                                                                                                                                   |

Reviewer: ```suggestion
| --force-counters32                              | Force to use 32-bit counters (even in snmp v2c and v3). Should be used when 64-bit counters are buggy.                                                                                                                                                                                   |
```

| --statefile-concat-cwd | If used with the '--statefile-dir' option, the latter's value will be used as a sub-directory of the current working directory. Useful on Windows when the plugin is compiled, as the file system and permissions are different from Linux.   |
| --statefile-format     | Define the format used to store the cache. Available formats: 'dumper', 'storable', 'json' (default).                                                                                                                                         |
| --statefile-key        | Define the key to encrypt/decrypt the cache.                                                                                                                                                                                                  |
| --statefile-cipher     | Define the cipher algorithm to encrypt the cache (default: 'AES').                                                                                                                                                                            |

> | --filter-*             | Filter name with regexp. Can be ('vdomain', 'vpn')                                                                                                                                                                                            |

Author: To split in plugin. Can we keep this in that state here at the moment?

| --statefile-concat-cwd   | If used with the '--statefile-dir' option, the latter's value will be used as a sub-directory of the current working directory. Useful on Windows when the plugin is compiled, as the file system and permissions are different from Linux.   |
| --statefile-format       | Define the format used to store the cache. Available formats: 'dumper', 'storable', 'json' (default).                                                                                                                                         |
| --statefile-key          | Define the key to encrypt/decrypt the cache.                                                                                                                                                                                                  |
| --statefile-cipher       | Define the cipher algorithm to encrypt the cache (default: 'AES').                                                                                                                                                                            |

> | --filter-vdomain         | Filter by virtual domain name (can be a regexp).                                                                                                                                                                                              |

Reviewer: unclear

> | --filter-vdomain         | Filter by virtual domain name (can be a regexp).                                                                                                                                                                                              |

Author: ```suggestion
| --filter-vdomain         | Define which virtual domains should be monitored based on their names. This option will be treated as a regular expression.                                                                                                                                                                                              |
```

> | --add-traffic            | Add traffic usage by virtual domain.                                                                                                                                                                                                          |

Reviewer: add?

Author: Without this option, you only monitor the state of the interface. With it, you collect the traffic statistics.

> | --add-policy             | Add number of policies by virtual domain.                                                                                                                                                                                                     |

Reviewer: add?

| --statefile-format       | Define the format used to store the cache. Available formats: 'dumper', 'storable', 'json' (default).                                                                                                                                         |
| --statefile-key          | Define the key to encrypt/decrypt the cache.                                                                                                                                                                                                  |
| --statefile-cipher       | Define the cipher algorithm to encrypt the cache (default: 'AES').                                                                                                                                                                            |
| --warning-* --critical-* | Thresholds. Can be: 'virus-detected', ''virus-detected-psec', 'virus-blocked', 'virus-blocked-psec'.                                                                                                                                          |

> | --filter-name            | Filter virtual domain name (can be a regexp).                                                                                                                                                                                                 |

Author: ```suggestion
| --filter-name            | Filter virtual domain name (can be a regexp).                                                                                                                                                                                                 |
```
```suggestion
| --filter-name            | Define which virtual domains should be monitored based on their names. This option will be treated as a regular expression.                                                                                                                                                                                                 |
```
